api/Window.json

```
@@ -4150,27 +4168,69 @@
                 "alternative_name": "clientInformation"
               }
             ],
-            "ie": {
-              "version_added": "4"
-            },
-            "opera": {
-              "version_added": "3"
-            },
-            "opera_android": {
-              "version_added": "10.1"
-            },
-            "safari": {
-              "version_added": "1"
-            },
-            "safari_ios": {
-              "version_added": "1"
-            },
-            "samsunginternet_android": {
-              "version_added": "1.0"
-            },
-            "webview_android": {
-              "version_added": "1"
-            }
+            "ie": [
+              {
+                "version_added": "4"
+              },
+              {
+                "version_added": true,
+                "alternative_name": "clientInformation"
+              }
+            ],
+            "opera": [
+              {
+                "version_added": "3"
+              },
+              {
+                "version_added": true,
```

```suggestion
                "version_added": "15",
```

Based on testing with http://mdn-bcd-collector.appspot.com/tests/api/Window/clientInformation